include/secure_services.h

```diff
@@ -15,13 +28,15 @@ extern "C" {
 #endif
 
 /** Request a system reboot from the Secure Firmware.
+ *
+ * Rebooting is not available from the Non-Secure Firmware.
```

Comment:
Maybe add a note that this API shouldn't be called directly but rather through the `sys_reboot` API? Or is this assumption wrong?

Reply:
Good point, done